timer.py

```diff
@@ -0,0 +1,69 @@
+import time
+
+
+class TimeError(Exception):
+    """
+    Errors on timer class.
```

Comment:
I think it is because that the only Exception that shouldn't jump when runing the program.
if she jump she should crush the program and the docstring is what will be wrote in the consle,
But this Exception happenet to me and i dont raise it so I will remove it. :face_with_head_bandage: